test/extern/jjueckstock/basic.chpl

```diff
@@ -1,3 +1,4 @@
+use CPtr;
```

Comment:
I don't have an LLVM-enabled build handy, so am curious whether this one is necessary or not?  Specifically, I believe that the use of an `extern` block results in an auto-insert of `use CPtr;`.  But maybe it's insufficient in some way that I'm not seeing?

(I'm asking out of curiosity over whether our attempt to be clever in inserting `use CPtr;` for extern blocks isn't working as intended once `CPtr` symbols are no longer auto-available).

Reply:
Nope, you are right. Removed this one, too.